definitions/types/azure-resource-id.json

```diff
@@ -6,7 +6,7 @@
   "examples": [
     "/subscriptions/12345678-1234-1234-abcd-1234567890ab/resourceGroups/resource-group-name/providers/Microsoft.Network/virtualNetworks/network-name"
```

Comment:
```suggestion
    "/subscriptions/12345678-1234-1234-abcd-1234567890ab/resourceGroups/resource-group-name/providers/Microsoft.Network/virtualNetworks/network-name",
    "/subscriptions/12345678-1234-1234-abcd-1234567890ab/resourcegroups/resource-group-name/providers/Microsoft.Network/virtualNetworks/network-name",
```

Reply:
I don't think this is needed. We currently aren't allowing customers to input their own resource ID. If we want to make an example for every resource ID variant, there would be at least 10 different examples.

Reply:
Okay, your call here. I thought it might be useful to capture one of the reasons for this code change (allowing uppercase G and lowercase) but no worries either way.